_categories/API-client.md

```diff
@@ -0,0 +1,6 @@
+---
+name: API-client
+title: API clients
+excerpt: Using the Synapse clients and REST API.
```

Comment:
Note that we discussed adding an `explanation:` section to provide in-depth context beyond a *catch phrase*.

Reply:
I like the idea of adding an explanation. Some of these seem obvious, but that might not always be the case. It may also help newbies to the project.